src/main/webapp/nickname.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Black Student Insight</title>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <header>
+      <h1>Black Student Insight</h1></a>
+    </header>
+    <main>
+      <h2>Enter a Name to Appear on All of Your Reviews</h2>
```

Comment:
A smaller nit, but I'd recommend this text to be in sentence case rather that title case. I think it reads uncomfortably with everything capitalized.